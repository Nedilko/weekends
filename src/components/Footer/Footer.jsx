function Footer() {
  return (
    <footer className="mb-2 text-center text-base font-thin">
      <a target="_blank" rel="noreferrer" href="https://nac.in.ua">
        nac.in.ua
      </a>
    </footer>
  )
}

export default Footer
