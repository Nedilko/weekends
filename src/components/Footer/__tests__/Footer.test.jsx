import Footer from '@components/Footer/Footer'
import { render, screen } from '@utils/test-utils'

describe('Footer', () => {
  it('should render link with text', () => {
    render(<Footer />)
    const footerLink = screen.getByRole('link')
    expect(footerLink).toBeInTheDocument()
    expect(footerLink).toMatchSnapshot()
  })
  it('should has attributes', () => {
    render(<Footer />)
    const footerLink = screen.getByRole('link')
    expect(footerLink).toHaveAttribute('href', 'https://nac.in.ua')
    expect(footerLink).toHaveAttribute('target', '_blank')
  })
})
