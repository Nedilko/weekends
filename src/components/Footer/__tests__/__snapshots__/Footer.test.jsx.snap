// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render link with text 1`] = `
<a
  href="https://nac.in.ua"
  rel="noreferrer"
  target="_blank"
>
  nac.in.ua
</a>
`;
